lib/floe/workflow/choice_rule/data.rb

```diff
@@ -12,7 +12,7 @@ def initialize(_workflow, _name, payload)
           super
 
           @variable = parse_path("Variable", payload)
-          parse_compare_key
+          parse_compare_key(payload)
```

Comment:
If the argument is that anything in `@payload` is duplicated by other ivars I get it, but the fact that we have to pass it in to other methods seems to be contrary to that

Reply:
we only pass at initialization time.
After that, we do not use it.